projects/KiKit/default.nix

```
@@ -0,0 +1,8 @@
+{pkgs, ...} @ args: {
+  packages =
+    {
+      inherit (pkgs) kikit;
+      inherit (pkgs.kicadAddons) kikit-library;
+    }
+    // {kicad-kikit = pkgs.kicadAddons.kikit;};
```

```suggestion
      kicad-kikit = pkgs.kicadAddons.kikit;
    };
```

No need to compute the attrset update

---

There's a `kikit` for both `pkgs` and `pkgs.kicadAddons`. Renaming the latter seemed the easiest approach. How would you call the `kikit` add-on?